client/src/pages/Reviews.js

```diff
@@ -28,12 +28,12 @@ const useStyles = makeStyles((theme) => ({
     margin: theme.spacing(1),
   },
   fabClose: {
-    position: "absolute",
+    position: "fixed",
```

Comment:
If u are using position 'fixed', u should make parent element position 'relative'. Otherwise fab button will we tied to the whole window, particularly, fab close button will be shown in the right bottom of the entire screen but not of the drawer